app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixAvailabilityBottomSheet.kt

```diff
@@ -66,6 +70,8 @@ fun QuickFixAvailabilityBottomSheet(
                               visible = true, onCloseRequest = { onDismissRequest() }),
                       config =
                           CalendarConfig(
+                              locale = Locale.UK,
```

Comment:
Since our App is in English, it's ok to set the locale to `UK` but consider changing it to `Locale.getDefault()` if we decide to push to prod ;)